widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/AdditionalDetailsPanel.java

```diff
@@ -0,0 +1,117 @@
+/*!
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2023-2024 Hitachi Vantara..  All rights reserved.
+ */
+
+package org.pentaho.mantle.client.dialogs.scheduling;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.ListBox;
+import com.google.gwt.user.client.ui.CheckBox;
+import com.google.gwt.user.client.ui.CaptionPanel;
+import com.google.gwt.user.client.ui.Label;
+import org.pentaho.gwt.widgets.client.controls.ErrorLabel;
+import org.pentaho.gwt.widgets.client.panel.HorizontalFlexPanel;
+import org.pentaho.gwt.widgets.client.panel.VerticalFlexPanel;
+import org.pentaho.gwt.widgets.client.ui.ICallback;
+import org.pentaho.gwt.widgets.client.ui.IChangeHandler;
+import org.pentaho.gwt.widgets.client.utils.TimeUtil;
+import org.pentaho.mantle.client.messages.Messages;
+
+public class AdditionalDetailsPanel extends Composite  implements IChangeHandler {
+
+    private ListBox logLevelLb = createLogLevelListBox();
+    private ICallback<IChangeHandler> onChangeHandler = null;
+    private ErrorLabel detailsLabel = null;
+    private CheckBox enableSafeMode;
+    private CheckBox gatherMetrics;
```

Comment:
Formatting still does not seem to follow the Pentaho coding standards — indentation is using 4 spaces, instead of 2.